applications/GeoMechanicsApplication/tests/cpp_tests/test_geo_mechanics_application.cpp

```diff
@@ -13,6 +13,10 @@
 #include "geo_mechanics_application.h"
 #include "geo_mechanics_fast_suite.h"
 
+#include <string>
+
+using namespace std::string_literals;
```

Comment:
What is the reason we add this line? Although this is a cpp file, I'd still prefer not to use using namespace std::, but that might be a personal preference

Reply:
Discussed, clear it's for brevity